torchio/transforms/augmentation/spatial/random_affine.py

```diff
@@ -104,7 +104,7 @@ def __init__(
             translation: TypeOneToSixFloat = 0,
             isotropic: bool = False,
             center: str = 'image',
-            default_pad_value: Union[str, float] = 'otsu',
+            default_pad_value: Union[str, float] = 'minimum',
```

Comment:
ah qu'est ce que ca fait la ...
-;)
merci

Reply:
Haha maybe this should go into a different PR, I'm mixing too much here. #344 should be addressed somewhere else!